packages/amplify_api_rest/android/build.gradle

```diff
@@ -0,0 +1,51 @@
+group 'com.amazonaws.amplify.amplify_api_rest'
+version '1.0-SNAPSHOT'
+
+buildscript {
+    ext.kotlin_version = '1.3.50'
```

Comment:
Latest stable release of Kotlin is 1.4.21.

Reply:
Thanks made a note to look into updating that in our merged branch.